pkg/filesystem/subvolume.go

```diff
@@ -175,8 +175,13 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 			}
 			// append the subvolume which doesn't have any snapshot attached to it.
 			for _, sv := range subvol {
-				state := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
-
+				state, err := getSubvolumeState(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name)
+				// subvolume info returns error in case of pending clone or if it is not ready
+				// it is suggested to delete the pvc before deleting the subvolume.
+				if err != nil {
+					logging.Info("Subvolume info not found for: %q", err)
+					logging.Info("Please delete the pending pv if any before deleting the subvolume %s", sv.Name)
```

Comment:
is it missing `continue` `return` or break ?

Reply:
yes, as per now it considers the subvolume as stale along with that it prints the above text, but yeah we can skip this subvolume.